File: JuegoAhorcado/JuegoAhorcadoBlazorServer/Services/PlayerSerializer.cs
using JuegoAhorcadoBlazorServer.Modelo;

namespace JuegoAhorcadoBlazorServer.Services
{
    public class PlayerSerializer
    {
        public string Serialize(Player player)
        {
            string values = "";

            values += player.Id;
            values += "%" + player.Name;
            values += "%" + player.Pass;
            values += "%" + player.Role;
            values += "%" + player.Rounds;
            values += "%" + player.Points;

            return values;
        }
    }
}

File: AhorcadoJuegoBackend/src/juegoUtils/PlayerSerializer.java
package juegoUtils;

import models.Player;
import services.Serializer;

public class PlayerSerializer implements Serializer{

	 @Override
	    public String serialize(Object t) {
	        try {
	            Player player = (Player) t;

	            String values = "";

	            values += player.getId();
	            values += "%" + player.getName();
	            values += "%" + player.getPass();
	            values += "%" + player.getRole();
	            values += "%" + player.getRounds();
	            values += "%" + player.getPoints();

	            return values;

	        } catch (Exception e) {
	            throw new RuntimeException(e.getMessage());
	        }
	    }
}
